dev/ci/docker_linux/Dockerfile

```diff
@@ -34,6 +35,7 @@ RUN bash "${NODEJS_INSTALL}/nodejs_install.sh"
 # Install the rest of the dependencies.
 RUN apt-get install -y --no-install-recommends \
   locales \
+  golang \
```

Comment:
add a comment here or below for the next reader?

Reply:
Done.